go/teams/chain.go

```diff
@@ -124,6 +124,49 @@ func (t TeamSigChainState) getUserRole(user keybase1.UserVersion) keybase1.TeamR
 	return role
 }
 
+func (t TeamSigChainState) AssertAdminRoleAt(uv keybase1.UserVersion, scl keybase1.SigChainLocation) (ret keybase1.SignatureMetadataBookends, err error) {
```

Comment:
Oh, I see. So rename the function? I can do that..  It's a pretty nice thing that likely we'll never need outside of the current caller...